app/models/team_member.rb

```diff
@@ -8,4 +8,28 @@ class TeamMember < ApplicationRecord
   def name
     first_name
   end
+
+  def job_title
+    if discipline == "Development"
+      "Developer"
+    elsif discipline == "Delivery"
+      "Delivery Lead"
+    elsif discipline == "Research"
+      "Researcher"
+    elsif discipline == "Service Design"
+      "Service Designer"
+    elsif discipline == "Design"
+      "Designer"
+    elsif discipline == "Operations Engineering"
+      "Operations Engineer"
+    elsif discipline == "Strategy"
+      "Strategist"
+    elsif discipline == "Technical Architecture"
+      "Technical Architect"
+    elsif discipline == "Product"
+      "Product Manager"
+    else
+      discipline
+    end
+  end
```

Comment:
I think this works great for the clear ones, like Developer, and Service Designer, but there are multiple roles inside the Strategy, for example. Not sure how best to handle that, but I think this is a good step forwards.